examples/enhanced_monitoring/README.md

```diff
@@ -0,0 +1,5 @@
+Enhanced Monitoring example
+===========================
+
+Configuration in this directory creates the additional resources required to use Enhanced Monitoring.
```

Comment:
Remember that these examples should always be executed using just terraform init, plan, apply. Users should not be asked to update anything in the example to try them out. Please include "provider" section and "rest of params here as per complete example".

Reply:
Are there guidelines for examples?

Reply:
No, but there will be (probably during next week or two).